spec/controllers/users_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
```

Comment:
Please use request specs, controller specs are deprecated in favour of the former